proposals/53:kread-start/eval.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Exit when any command fails
+set -e
+
+source /usr/src/upgrade-test-scripts/env_setup.sh
+
+ls -al
+
+yarn install --frozen-lockfile
+
+# XXX to avoid rebuilding lower layers
+# TODO clean up JS lib code so it's more independent
+cp package.json yarn.lock /usr/src/upgrade-test-scripts/lib/
+cd /usr/src/upgrade-test-scripts/lib/ && yarn install --frozen-lockfile
+cd -
+
```

Comment:
obsolete with the recent Yarn v4 change. There are some other straggles too
```suggestion

```

Reply:
Hm. How does yarn 4 make this obsolete? What does it do differently?

Reply:
(repeating our offline convo)

It's not Yarn v4 per se but the PR which introduced it. https://github.com/Agoric/agoric-3-proposals/pull/17/commits/fdbd756cc02cf479b7785a1be10f4c87a9ebaa1d